composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
+        "doctrine/annotations": "^1.2",
```

Comment:
`annotations` + required by it `lexer` packages are both very light. let us keep it in `requirements`.